pkg/fsm/fsm_active_file_io.go

```diff
@@ -95,6 +95,44 @@ func (f *FsMachine) saveFile(file *types.InputFile) StateFn {
 	return activeState
 }
 
+func (f *FsMachine) statFile(file *types.StatFile) StateFn {
+
+	// create the default paths
+	sourcePath := fmt.Sprintf("%s/%s/%s", file.SnapshotMountPath, "__default__", file.Filename)
```

Comment:
there's a method to do this without templates: `filepath.Join()`

Reply:
There is indeed! I shameless cribbed this bit from the other functions in this file without improving any of them, but I can fix that now...